### PR TITLE
git town 0.8.0

### PR DESCRIPTION
**Pull request by charlierudolph**

### CODE REVIEW

File: Library/Formula/git-town.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GitTown < Formula
   desc "High-level command-line interface for Git"
   homepage "http://www.git-town.com"
-  url "https://github.com/Originate/git-town/archive/v0.7.3.tar.gz"
-  sha256 "947fd6dac603dcb02467cc2969ef26db1ccdc72312b6b0d70fb8d11179eb09ba"
+  url "https://github.com/Originate/git-town/archive/v0.8.0.tar.gz"
+  sha256 "8ac3e38c04087757424bd58a431c78c9e16e9522"
```
Comment (bfontaine) [inline Library/Formula/git-town.rb]: This doesn’t look like a `sha256`.

Comment (charlierudolph) [inline Library/Formula/git-town.rb]: Indeed that was a sha1. Sorry, it has been updated

File: Library/Formula/git-town.rb [hunk 2]
```diff
@@ -11,8 +11,6 @@ class GitTown < Formula
     sha256 "09560a9c1c363f097b8a06d2ff86f60bcdbfe592e848fb61cf0a89d1bcd2c727" => :mountain_lion
   end
 
-  depends_on "dialog" => :recommended
```
Comment (mikemcquaid) [inline Library/Formula/git-town.rb]: This can't/isn't used even optionally any more? Just wanted to double-check.

Comment (charlierudolph) [inline Library/Formula/git-town.rb]: Correct we removed the command that required this as a dependency.

**Comment by bfontaine** [comment]
Thank you for your contribution @charlierudolph; we really appreciate it! :tada:

**Pull request closed without merging**